routes/dashboard/portals/feedbacks-24x7/index.js

```diff
@@ -6,7 +6,7 @@ let router = express.Router();
 let feedbacksModel = fq("schemas/feedbacks");
 let feedbacks = require("./../feedbacks");
 
-["/", "/step-1"].forEach(step => {
+["/", "/step-1"].forEach((step) => {
```

Comment:
Was this change done by prettier automatically?

Reply:
yeah